plugins/cortex_v2/icon_cortex_v2/util/util.py

```diff
@@ -0,0 +1,72 @@
+from typing import List, Dict
+
+
+def report_to_dict(report):
+    return report.json()
+
+
+def analyzers_to_dicts(analyzers):
+    return _convert_to_dicts(
+        analyzers,
+        ("id", "version", "dataTypeList", "name", "description", "license", "author", "url"),
+    )
+
+
+def job_to_dict(job, api):
+    job = _convert_to_dict(job, ("status", "date", "id", "analyzerId"))
+    job["artifacts"] = _artifacts_to_dicts(api.jobs.get_artifacts(job["id"]))
+    return job
+
+
+def jobs_to_dicts(jobs, api):
+    return [job_to_dict(j, api) for j in jobs]
```

Comment:
Please add type-hints 😃 
```suggestion
def jobs_to_dicts(jobs, api):
    return [job_to_dict(job, api) for job in jobs]
```